src/app/toolbar/toolbar.component.html

```diff
@@ -26,9 +26,12 @@
              *ngIf="viewTemplate !== undefined || (config.views)">
           <ng-template [ngTemplateOutlet]="viewTemplate" [ngTemplateOutletContext]="{}"></ng-template>
           <span *ngIf="config.views">
-            <button *ngFor="let view of config.views" class="btn btn-link"
+            <button *ngFor="let view of config.views"
+                    class="btn btn-link"
                     [ngClass]="{'active': isViewSelected(view), 'disabled': view.disabled === true}"
-                    title={{view.tooltip}}  (click)="viewSelected(view)">
+                    [attr.aria-label]="view.tooltip"
+                    title={{view.tooltip}}
```

Comment:
Instead of reusing the tooltip, should we specify a separate ariaLabel property in ToolbarView. We can always use the tooltip as a default.

Whatever we do, lets try to be consistent with the copy component. I believe we introduced separate ariaLabel properties there, but we can use tooltip for all components if that's sufficient?

Reply:
Can you clarify what controls you're considering doing this for?

When you say `title` is this the tooltip component, or the `title` attribute?